asm/internal/ll.bnf

```diff
@@ -1770,7 +1776,7 @@ CallConv
 	| "spir_kernel"
 	| "swiftcc"
 	| "webkit_jscc"
-	| "x86_64_sysvcc"
+	| "x86_64_sysvcc"      << ast.CallConvX86_64SysV, nil >>
```

Comment:
The full list of calling conventions were added in rev f0c1093fae039472e8f4186a80850252d02583e2. Sorry, I had forgotten to push this to GitHub.